### ISSUE TITLE
PluginException

### ISSUE THREAD

**Issue by xarrek** [klesun/deep-assoc-completion#61]
Hello,
I get this error on start.

PhpStorm 2018.3.2
Build #PS-183.4886.46, built on December 19, 2018
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

```
com.intellij.diagnostic.PluginException: com/intellij/lang/javascript/JavascriptLanguage [Plugin: lv.midiana.misc.phpstorm-plugins.deep-keys]
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.convertStub(ActionManagerImpl.java:149)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getActionImpl(ActionManagerImpl.java:471)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.getAction(ActionManagerImpl.java:459)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.preloadActions(ActionManagerImpl.java:1275)
	at com.intellij.openapi.actionSystem.impl.ActionPreloader.preload(ActionPreloader.java:32)
	at com.intellij.openapi.application.Preloader.lambda$null$0(Preloader.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.application.Preloader.lambda$initComponent$1(Preloader.java:63)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:227)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$100(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2$1.run(BoundedTaskExecutor.java:200)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:229)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoClassDefFoundError: com/intellij/lang/javascript/JavascriptLanguage
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.convertStub(ActionManagerImpl.java:145)
	... 19 more
Caused by: java.lang.ClassNotFoundException: com.intellij.lang.javascript.JavascriptLanguage PluginClassLoader[lv.midiana.misc.phpstorm-plugins.deep-keys, 2018.12.26.002] com.intellij.ide.plugins.cl.PluginClassLoader@72b3d1d4
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:49)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 22 more
```

**Comment by klesun** [comment]
Oh, thanks, sorry. Looks like you don't have a Javascript plugin. Did not consider one could have PHP plugin without Javascript... I'll fix this in a hour or so.

**Comment by xarrek** [comment]
I have "JavaScript Support" plugin. What exactly is the plugin about?

**Comment by klesun** [comment]
I'm not certain how this happens, but `JavascriptLanguage` is a class defined in the `Javascript` plugin, and since there is a `ClassNotFoundException` in stack trace I concluded that `Javascript` plugin is disabled or something. Though I'm not sure if it is actually disablable in phpstorm...

Nevertheless, `JavascriptLanguage` is used in just one specific place for syntax highlighting in a popup, so  I guess, I'll just remove it to fix the issue.

**Comment by klesun** [comment]
This update should fix your issue: [2018.12.28.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/53620)
